pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -47,7 +48,12 @@ func (s *AWSCloud) ensureLoadBalancer(name string, listeners []*elb.Listener, su
 
 		createRequest.SecurityGroups = stringPointerArray(securityGroupIDs)
 
-		glog.Info("Creating load balancer with name: ", name)
+		createRequest.Tags = []*elb.Tag{
+			{Key: aws.String(TagNameKubernetesCluster), Value: aws.String(s.getClusterName())},
+			{Key: aws.String("KubernetesService"), Value: aws.String(namespacedName.String())},
```

Comment:
I think we should keep tag name in the kubernetes.io namespace (I screwed up on the format of TagNameKubernetesCluster;).  How about `kubernetes.io/service-name`?  (And I suggest making it a constant so we group these magic tags together).